src/app/services/profile/profile.service.ts

```diff
@@ -13,7 +13,9 @@ import { APPCONFIG } from 'src/app/config';
 })
 export class ProfileService {
   profileUrl = '/auth/profile/';
+  depositeUrl = '/transactions/';
```

Comment:
Hey @shemaeric, we need to find more meaningfully url structures, we could brainstorm ideas over a sync.